sets/hinterlands.py

```diff
@@ -255,6 +257,33 @@ def play(self, skip=False):
 		crd.Card.on_finished(self, True)
 
 
+class Ill_Gotten_Gains(crd.Money):
+	def __init__(self, game, played_by):
+		crd.Money.__init__(self, game, played_by)
+		self.title = "Ill Gotten Gains"
+		self.description = "Worth {}\nWhen you play this, you may gain a copper, putting it in your hand." \
+		                   "When you gain this, each other player gains a Curse".format(crd.format_money(1))
+		self.value = 1
+		self.price = 5
+		self.type = "Treasure"
+
+	@gen.coroutine
+	def play(self, skip=False):
+		crd.Card.play(self, skip)
+		self.played_by.balance += self.value
+		self.played_by.update_resources(True)
+
+		choice = yield self.played_by.select(1, 1, ["Yes", "No"], "Gain a Copper to hand?")
+		if choice[0] == "Yes":
+			yield self.played_by.gain_to_hand("Copper")
+		crd.Card.on_finished(self, True)
+
+	@gen.coroutine
+	def on_gain(self):
+		for i in self.played_by.get_opponents():
```

Comment:
you could make this one line `for opponent in self.played_by.get_opponents(): yield(opponent.gain("Curse"))`